src/app/signup/page.tsx

```diff
@@ -102,14 +193,20 @@ const SignUp = () => {
                       {errors.password.message}
                     </p>
                   )}
+                {passwordError && (
+                  <p className="text-red-500 text-sm mt-1">{passwordError}</p>
+                )}
               </div>
               <div className="mb-4">
                 <input
                   type="password"
+                  autoComplete="new-password"
                   placeholder="Confirm Password"
-                  maxLength={64}
-                  className="w-full bg-gray-300 text-black size-12 placeholder:text-black border px-4 rounded"
+                  maxLength={SIGNUP_MAX_PASSWORD_LENGTH}
+                  minLength={SIGNUP_MIN_PASSWORD_LENGTH}
```

Comment:
Isn't it for semantic purposes? Or am I wrong? I've always thought they held semantic meaning.

Reply:
What do you mean by semantics? My understanding was that this is what the HTML validator uses. But, since we have `noValidate` (ie: turned off HTML validations) this shouldn't be necessary anymore.

Reply:
I see. I guess I'll remove it.